src/Evented.ts

```diff
@@ -36,6 +36,22 @@ function handlesArraytoHandle(handles: Handle[]): Handle {
 	};
 }
 
+export interface EventObject {
+	/**
+	 * The type of the event
+	 */
+	readonly type: string | symbol;
```

Comment:
`@dojo/core/on` uses it, but it seems like everyone else doesn't specifically refer to that interface directly when writing overloads for other `.on()` methods for custom events.  Historically, it had cross cutting concerns when `on()` and `Evented` were in separate packages.